plugins/plotly-express/src/deephaven/plot/express/preprocess/StyleManager.py

```diff
@@ -31,16 +32,25 @@ def __init__(
         self.cycled = cycle(self.ls)
         self.found = {}
 
-    def assign_style(self, val: str) -> str:
+        self.wildcard = None
+
+    def assign_style(self, val: str, map_applies: bool) -> str:
         """
         Assign and return a style for the specified value.
 
         Args:
             val: The val to assign
+            map_applies: Whether the map applies to this value
 
         Returns:
             The assigned style
         """
+
+        if not map_applies:
+            if self.wildcard is None:
+                self.wildcard = next(self.cycled)
+            # This should always be a string but pyright complains
+            return str(self.wildcard)
```

Comment:
Instead of casting it to a str, assert that it's not `None`:
```suggestion
            assert self.wildcard is not None
            return self.wildcard
```
Since it should always be a string.